psi4/src/psi4/lib3index/df_helper.h

```diff
@@ -47,50 +46,62 @@ class Matrix;
 class ERISieve;
 class TwoBodyAOInt;
 
-namespace df_helper{
+namespace df_helper {
 
 class DF_Helper {
-
-public:
-
+   public:
     DF_Helper(std::shared_ptr<BasisSet> primary, std::shared_ptr<BasisSet> aux);
     ~DF_Helper();
 
-    // user options, must set before calling build()---------------------------------
-    // memory in doubles
-    void set_memory(size_t mem) { memory_ = mem;}
-    size_t get_memory() { return memory_; }
-
-    // workflow method (store or direct)
-    void set_method(std::string met) { method_ = met;}
+    // => user options - set these before calling initialize() <=
+    // workflow method (store or direct) (defaults to STORE)
```

Comment:
I know we're horribly inconsistent with Doxygen in the rest of the code, but could you use the `///` comment marker instead of `//`, please?  That will trigger this class to be Doxygened, which could be useful for developers.